### ISSUE TITLE
making the lr backpacks aquire their id

### ISSUE THREAD

**Issue by kripto202** [michail-nikolaev/task-force-arma-3-radio#1145]
I'm trying to create radio backpacks for my group but I keep coming accross this problem where the backpack resets when you access the Virtual Arsenal. I looked over the code and I can't seem to find anything to state the lr backpack id. 

I did find this and added the radio_ids.hpp file into my mod. https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/master/arma3/%40task_force_radio/addons/task_force_radio_items/config.cpp#L554

**Comment by kripto202** [comment]
if you need my config, let me know

**Comment by dedmen** [comment]
radio_ids.hpp is only used for handheld radios.
Backpacks have their ids stored in their variables, Does that problem also happen with the TFAR backpack radios?
Yeah I'd like to take a look at your config.

**Comment by kripto202** [comment]
it only happens on my backpacks

config.cpp http://pastebin.com/0Z6kfZtZ
cfgweapons.hpp http://pastebin.com/XuiLRhYd
cfgvehicles.hpp http://pastebin.com/kEPVFf5y

**Comment by CorruptedHeart** [comment]
Backpacks are objects within arma. So there is no need for IDs like items. What is likely happening is Virtual Arsenal is removing and adding a new backpack which would not have the settings from the old one.

**Comment by kripto202** [comment]
is there a possible way to fix that?

**Comment by dedmen** [comment]
Again the question. Does that also happen to the TFAR original backpacks? I can't remember that it did.

**Comment by kripto202** [comment]
will need to check

**Comment by kripto202** [comment]
ya it affects the tfar backpacks

**Comment by dedmen** [comment]
Can you please provide the steps to reproduce that? Is just opening arsenal and closing it without changing anything enough? Do you put some item in your backpack? What exactly do you do step by step

**Comment by kripto202** [comment]
steps to reproduce, load the backpack through VA, change the channel, 45, the go back into the arsenal. When I go back in, without changing anything, I go back out and check the LR backpack. I would find the LR radio freq changed.

**Comment by dedmen** [comment]
#970 #884 duplicates

**Comment by dedmen** [comment]
We have an eventhandler on arsenal close
```
[missionnamespace,"arsenalClosed",[displaynull,uinamespace getvariable ["BIS_fnc_arsenal_toggleSpace",false]]] call bis_fnc_callscriptedeventhandler;```

**Comment by dedmen** [comment]
Fixed in #1150
